packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/UpgradeGuides.f/v7.0.0.s.tool.mdx

```diff
@@ -0,0 +1,150 @@
+|`appConfig.defaultDirection`|`defaultDirection`|No longer nested under appConfig.|
+|`appConfig.extensions`|`extensionItems`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.iconPath`|`extensionItems.iconPath`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.key`|`extensionItems.key`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.text`|`extensionItems.text`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.componentPath`|`extensionItems.modalFilePath`| Renamed to prepare for application v2 modals.|
+|`appConfig.extensions.size`|**removed**|One size fits all.|
+|`appConfig.headHtml`|`headHtml`|No longer nested under appConfig.|
+|`includeTestEvidence`|**removed**|Removed the ability to include test evidence, this should be covered now by a separate site report.|
+
+## Dependency changes
+
+Previously `terra-dev-site` was providing a version of `rect-intl` as a dependency. Because this satisfied the `terra-application` peer dependency requirement, many applications did not define it. With this release we have removed that dependency and applications will have to provide that peer dependency themselves.
+
+`terra-application` is now a peer dependency and must be defined by the package consuming `terra-dev-site`.
+
+`terra-toolkit` is no longer a peer dependency to allow consumers to use `terra-dev-site` v7 with either terra-toolkit or the replacement `terra-functional-testing` package.
+
+### Resulting Test changes
+
+The page introduces a new tab stop. Any tests using tabstops will have to be updated to include an additional tab stop.
+
+If you happen to be testing md or mdx components, a wrapping div has been removed and screenshots will have to be regenerated. I assume this is rare beyond `terra-dev-site` itself.
+
+## Node 10
+
+Node 10 is now the minium supported node version. This was bumped to support version 5 of react-docgen, which is used to generate props tables.
+
+## MDX
+
+The MDX loader is now used to load both `.md` and .`mdx` files. This may cause build errors in existing .md documents. This change allows us to remove the marked loader from the plugin.
+
+A couple of common syntax errors when using the MDX loader can be found on our [Writing Documentation](/dev_tools/cerner-terra-application-docs/terra-dev-site/writing-documentation#mdx-loader) Page.
+
+## I get the webpack error: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
+
+You can get this error when using webpack 4. Part of the terra-dev-site webpack plugin we inject a entry to build the site. Webpack requires an entry to function but when the check is performed changed between webpack 4 and 5. In webpack 4 and entry is checked prior to running the plugin where as in webpack 5 the check is performed after.
```

Comment:
```suggestion
You can get this error when using webpack 4. Part of the `terra-dev-site` webpack plugin we inject an entry to build the site. Webpack requires an entry to function but the check that is performed changed between webpack 4 and 5. In webpack 4, an entry is checked prior to running the plugin, where as in webpack 5 the check is performed after.
```
This automatic entry point injection was removed in v7 right? Does the reference of the plugin need to specify v6? Also is the entrypoint needed at all in webpack v5 based on other comments I read or do we inject it after the webpack check which is why there is an error?

Reply:
Automatic entypoint injection was added with v7 and is done regardless of webpack version, adding the dummy entrypoint is only needed when using webpack 4.